pyanaconda/anaconda.py

```diff
@@ -91,7 +91,14 @@ def bootloader(self):
     def instClass(self):
         if not self._instClass:
             from pyanaconda.installclass import factory
-            self._instClass = factory.get_base_install_class()
+
+            # Get required install class.
+            if self.ksdata.installclass.seen:
+                name = self.ksdata.installclass.name
+                self._instClass = factory.get_required_install_class(name)
+            # Or just find the best one.
```

Comment:
I wouldn't name `base` install class as "the best one"; or the function name below is incorrect.